src/__experimental__/components/search/search.component.js

```diff
@@ -17,9 +17,25 @@ const Search = ({
   );
   let inputRef = useRef(null);
   const [searchValue, setSearchValue] = useState(initialValue);
-  const [isActive, setIsActive] = useState(inputRef.current === document.activeElement);
+  const [isFocused, setIsFocused] = useState(inputRef.current === document.activeElement);
```

Comment:
I think there's no need for this `inputRef.current === document.activeElement` comparison as `inputRef.current` is undefined when `useState` is initialized anyway

Reply:
you're right, removed